CppCoreGuidelines.md

```diff
@@ -749,7 +749,7 @@ The date is validated twice (by the `Date` constructor) and passed as a characte
 ##### Example
 
 Excess checking can be costly.
-There are cases where checking early is dumb because you may not ever need the value, or may only need part of the value that is more easily checked than the whole.
+There are cases where checking early is dumb because you may not ever need the value, or may only need part of the value that is more easily checked than the whole.  Similarly, don't add validity checks that change the asymptotic behavior of your interface (e.g., don't add a `O(1)` check to an interface with an average complexity of `O(1)`).
```

Comment:
1) I'm missing the distinction beween `O(1)` and `O(1)`. :-S

2) "Similarly" is add odds with the grammatical structure here; the first sentence is in the indicative, the second is an imperative.